tools/oecert/host/host.cpp

```diff
@@ -469,7 +554,8 @@ static oe_result_t _process_params(oe_enclave_t* enclave)
     // "SGX_AESM_ADDR" to "1" and unset after process finishes
     if (sgx_asem_env == nullptr)
     {
-        if (strcmp(INPUT_PARAM_QUOTE_OUT_OF_PROC, _params.quote_proc) == 0 &&
+        if (strcasecmp(INPUT_PARAM_QUOTE_OUT_OF_PROC, _params.quote_proc) ==
```

Comment:
It just came to me that since we only have 3 possible values for `quote_proc`, ("null", "in", "out"), we can consider switch it from `string` to `int` so we can avoid `strcasecmp` here. Just a thought.